internal/actions/pr.go

```diff
@@ -671,97 +670,78 @@ type PRMetadata struct {
 	Trunk      string `json:"trunk"`
 }
 
-const PRMetadataCommentStart = "<!-- av pr metadata\n"
```

Comment:
I removed the newline here because I noticed when I'd edit the PR manually on GitHub, I'd end up with a carriage return, i.e., `"<!-- av pr metadata\r\n"` and this parsing would break. Instead, opting to just parse this directly and add any extra whitespace padding explicitly.